.github/workflows/test-e2e-windows.yml

```
@@ -290,3 +299,9 @@ jobs:
           path: test-logs
           if-no-files-found: ignore
 
+      - name: Fail workflow if any test step failed
+        if: |
+          steps.extensions.outputs.exit_code != '0'
+        run: |
+          echo "Extension tests failed."
```

```suggestion
      - name: Fail workflow if extension tests failed
        if: |
          steps.extensions.outputs.exit_code != '0'
        run: |
          echo "Extension tests failed."
```

This isn't about "any test step"